src/Features/Core/Portable/AddParameter/AddParameterService.cs

```diff
@@ -0,0 +1,344 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeGeneration;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+
+namespace Microsoft.CodeAnalysis.AddParameter
+{
+    internal class AddParameterService
+    {
+        private AddParameterService() { }
```

Comment:
in general, we still always wrap these :)

Reply:
As In?

Reply:
```c#
private AddParameterService()
{
}
```

Reply:
This is correct in general, but not required. The requirement has been explicitly disabled here:
https://github.com/dotnet/roslyn/blob/d79e5f2ce4d2d74d2c34afdfe40953d9c7caf69a/.editorconfig#L199